src/lib/utils.ts

```diff
@@ -48,7 +48,7 @@ export const getFormattedRows = (
       withDeltaSize(report.targetBranchReport[fileName], report.currentBranchReport[fileName])
     ])
     .map(([fileName, targetBranchSize, currentBranchSize]) => [
-      fileName.replace(omitFromFilename, ''),
+      fileName.slice((fileName.match(stampFromMatch) || ({} as any)).index),
```

Comment:
we don't have `workdir` from the cli, so we need to match the dist from the full path and remove what before that

Reply:
this won't be needed if #59 is merged